WeakAuras/Prototypes.lua

```diff
@@ -4715,8 +4715,9 @@ WeakAuras.event_prototypes = {
         "UNIT_SPELLCAST_INTERRUPTED"
       };
       AddUnitChangeEvents(trigger.unit, result)
-      if trigger.target and trigger.target ~= "" then
+      if trigger.use_destUnit and trigger.destUnit and trigger.destUnit ~= "" then
```

Comment:
What does this fix, exactly? I’m not familiar with this but of code.

Reply:
trigger.target doesn't exists, "Caster's Target" option is in trigger.destUnit
I guess at one point i renamed it and forget to edit it here

If trigger needs to know who is the target of trigger.unit then 
  register event "UNIT_TARGET" to know when target/focus/boss1/... change target

ps: i forced-push removal of AddUnitChangeEvents on trigger.destUnit